### PR TITLE
updated possibility to ingest notifications using approvals flow

### PR DESCRIPTION
**Pull request by denyso-bb**
Updated possibility to ingest notifications using approvals flow that was implemented in 2.16.0, since service layer was restricted to create global notifications

### CODE REVIEW

File: src/main/java/com/backbase/ct/bbfuel/configurator/ApprovalsConfigurator.java [hunk 1]
```diff
@@ -209,6 +226,15 @@ private void assignContactsPolicies(String externalServiceAgreementId) {
             singletonList(createPolicyAssignmentRequestBounds(policyAId, null))));
     }
 
+    private List<IntegrationPolicyAssignmentRequest> getNotificationsPolicyAssignmentsBasedOnZeroApprovalPolicyOnly(
+        String externalServiceAgreementId) {
+        return singletonList(createPolicyAssignmentRequest(
+            externalServiceAgreementId,
+            NOTIFICATIONS_RESOURCE_NAME,
+            NOTIFICATIONS_FUNCTION_NAME,
+            singletonList(createPolicyAssignmentRequestBounds(policyZeroId, null))));
```
[the remark this answers is not in the document]
Comment (denyso-bb) [reply src/main/java/com/backbase/ct/bbfuel/configurator/ApprovalsConfigurator.java]: Actually, this is just a fix for global notifications setup, as a first step - to fix what was working before the breaking changes.
But yes, further we will require more complex policies and some "pending" (awaiting approval) notifications presetup for demo purposes. So notifications will probably use both zero approvals and single approval policies.

File: docs/CAPABILITY_DATA_SETUP.md [hunk 2]
```diff
@@ -68,4 +68,21 @@ Periodic limit per service agreement, for function group "Admin":
 
 - Transactional limit with amount of 1,000,000
 - Privileges "create" and "approve"
-- Functions SEPA CT, US Domestic Wire and US Foreign Wire
\ No newline at end of file
+- Functions SEPA CT, US Domestic Wire and US Foreign Wire
+
+## Notifications setup
+Global notifications are only created be the bank manager using approvals
```
Comment (richard-blank) [inline docs/CAPABILITY_DATA_SETUP.md]: created **by**

Comment (denyso-bb) [reply docs/CAPABILITY_DATA_SETUP.md]: thanks! fixed in commit: https://github.com/Backbase/bb-fuel/pull/30/commits/96933fcec957405afb3b0ad85a9ce08e0dce2272

File: jenkins/bb-fuel.groovy [hunk 3]
```diff
@@ -52,6 +53,7 @@ pipeline {
                                     "-Duse.pfm.categories.for.transactions=${params.USE_PFM_CATEGORIES_FOR_TRANSACTIONS} " +
                                     "-Dingest.approvals.for.payments=${params.INGEST_APPROVALS_FOR_PAYMENTS} " +
                                     "-Dingest.approvals.for.contacts=${params.INGEST_APPROVALS_FOR_CONTACTS} " +
+                                    "-Dingest.approvals.for.contacts=${params.INGEST_APPROVALS_FOR_NOTIFICATIONS} " +
```
Comment (richard-blank) [inline jenkins/bb-fuel.groovy]: ingest.approvals.for.**notifications**=

Comment (denyso-bb) [reply jenkins/bb-fuel.groovy]: yes, that was my copy / paste issue, thank you!

**Pull request merged**